vowpalwabbit/core/include/vw/core/estimators/cressieread_estimator.h

```diff
@@ -8,6 +8,8 @@
 
 namespace VW
 {
+namespace estimators
+{
 class cressieread_estimator
```

Comment:
super nit: this class could be named simply `cressieread` now

Reply:
true might as well